guess_a_no/README.md

```diff
@@ -0,0 +1,37 @@
+Game: Guess A Number
+=======
+
+### Objective
+
+Objective is to build a fun game, which will help a User to get familar with random no and basic logic. It's a small script and any language can be used for coding.
+

```

Comment:
This is simple enough for the user to understand and start coding, adding any sort of theme will complicate, fair enough It may spike up his interest but graphics will be preferred if we are asking him to code in JS, which we are not.